App/Containers/Example/ExampleScreen.js

```diff
@@ -61,6 +63,7 @@ class ExampleScreen extends React.Component {
               onPress={() => this._fetchUser()}
               title="Refresh"
             />
+            <ProgressBar progress={0.34}/>
```

Comment:
We want to pass the actual progress not as a prop but as something through react redux. For now store the progress into state instead.